tracer/src/Datadog.Trace/IAST/StackWalker.cs

```diff
@@ -13,7 +13,19 @@ namespace Datadog.Trace.Iast;
 internal static class StackWalker
 {
     public static readonly string[] ExcludeSpanGenerationTypes = { "Datadog.Trace.Debugger.Helpers.StringExtensions" };
-    public static readonly string[] AssemblyNamesToSkip = { "Datadog.Trace", "System.Security.Cryptography", "System.Security.Cryptography.Primitives", "System.Security.Cryptography.Algorithms", "System.Security.Cryptography.Csp" };
+    public static readonly string[] AssemblyNamesToSkip =
+    {
+        "Datadog.Trace",
+        "System.Security.Cryptography",
+        "System.Security.Cryptography.Primitives",
+        "System.Security.Cryptography.Algorithms",
+        "System.Security.Cryptography.Csp",
+        "System.Data.SQLite",
```

Comment:
What about `Microsoft.Data.SqlClient` and `System.Data.SqlClient`?

Reply:
You are totally right. I have reviewed the list of instrumented assemblies and added them all (I thought that I already had done that, but clearly, I missed something).  Some others assemblies were missing as well. I am preparing a separate instrumented test PR that would allow us to test all the cases without the need to create new endpoints or sample apps.